testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/AbstractCorsExampleAdapterTest.java

```diff
@@ -39,6 +40,7 @@
 /**
  * Created by fkiss.
  */
+@Ignore //Needs a discussion about future work.
```

Comment:
Can you elaborate on why this is ignored? Also is there a JIRA for "future work"?